ydb/docs/en/core/concepts/backup-collections.md

```diff
@@ -0,0 +1,308 @@
+# Backup collections
+
+Backup collections provide an advanced backup solution for {{ ydb-short-name }} that organizes full and incremental backups into managed collections. This approach is designed for production workloads requiring efficient disaster recovery and point-in-time recovery capabilities.
```

Comment:
The second sentence doesn't explain how these claims are achieved. If this is indeed a schema object, give a high-level explanation of how it helps recovery if the whole cluster is gone for whatever reason. Or is the main use case recovery of like individual tables removed due to human error? Again, state this in the beginning so it's clear what kind of backups we're talking about and what to expect.